src/document/json/counter.ts

```diff
@@ -90,6 +90,7 @@ export class Counter {
       );
     }
 
+    this.counter.increase(value);
```

Comment:
According to [Yorkie documentation](https://github.com/yorkie-team/yorkie/blob/main/design/document-editing.md), value in `this.counter`(CRDT) must be manipulated through `LocalChanges` buffer, not be changed in `Counter` object directly.